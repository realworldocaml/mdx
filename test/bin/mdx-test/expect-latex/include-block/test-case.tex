We include files with MDX comments:

% $MDX file=code.mli

% $MDX file=dune

% $MDX file=../.ocamlformat
