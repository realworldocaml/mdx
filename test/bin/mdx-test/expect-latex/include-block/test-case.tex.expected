We include files with MDX comments:

% $MDX file=code.mli
\begin{ocaml}
type t

val empty : t
\end{ocaml}

% $MDX file=dune
\begin{scheme}

\end{scheme}

% $MDX file=../.ocamlformat
\begin{}
disable=true
\end{}
